recipes-ni/initscripts-nilrt/initscripts-nilrt_1.0.bb

```diff
@@ -14,8 +14,7 @@ RDEPENDS_${PN}-ptest += "bash"
 DEPENDS_append_x64 = " nilrtdiskcrypt "
 RDEPENDS_${PN}_append_x64 = " nilrtdiskcrypt "
 
-SRC_URI = "file://nisetbootmode \
-           file://firewall \
+SRC_URI = "file://firewall \
```

Comment:
As long as you're touching this line, could you just refactor it like this, so that it stops showing up in the diff?

```
SRC_URI = " \
    file://firewall \
```

Reply:
Done